classes/dfs/amazon.php

```diff
@@ -36,13 +36,16 @@ public static function build()
         $ini = eZINI::instance( 'dfsamazons3.ini' );
         $parameters = $ini->group( "BackendSettings" );
 
-        $options = array(
-            'credentials' => [
-                'key' => $parameters['AccessKeyID'],
-                'secret' => $parameters['SecretAccessKey'],
-            ],
-            'version' => 'latest'
-        );
+        if(isset($parameters['AccessKeyID'], $parameters['SecretAccessKey'])) {
```

Comment:
Hi, thanks for contributing!

Q: Why did you add the` isset` here when you did not for `Version`?

Reply:
ok, so then maybe change this line to the following to make it more clear you are checking if it has value or not:
```suggestion
        // Only set credentials if they have been configured in INI, if not client will attempt to load them from environment instead.
        if($parameters['AccessKeyID'] && $parameters['SecretAccessKey']) {
```
And maybe add in the ini something like the following before AccessKeyID:
```
# These maps to client credentials "key" & "secret". If not set, the SDK will attempt to load them from the environment.
```

Also, should default value of `Version` in INI be set to `latest` like before? If this is implicit the same as empty, then let's add that to the suggestion for Version INI doc as well.